vignettes/programming_strategy.Rmd

```diff
@@ -739,6 +739,17 @@ Please take the following list as recommendation and try to adhere to its rules
 * `dplyr::if_else()` should be used when there are only two conditions. 
 Try to always set the `missing` argument whenever appropriate.
 
+## How Quoting is used
+* Some admiral arguments require selecting one particular option like `mode`, e.g. `mode = "last"`. Use quotation marks to capture these. The expected assertion function corresponding to these arguments is `assert_character_scalar()/assert_character_vector()`.
+* Many admiral arguments require capturing an expression which are to be evaluated _later_ inside the function body, see arguments like `new_vars`, e.g. `new_vars = exprs(TRTSDTM = EXSTDTM)`. Oftentimes, the assertion function corresponding to these are `assert_expr()/assert_expr_list()`
+* Use `!!` or `!!!` to unquote the expression(s) as needed. Keep in mind `!!` is a one-to-one replacement and `!!!` is a one-to-many replacement. Please see [this chapter](https://adv-r.hadley.nz/quasiquotation.html) in the Advanced R textbook for more details.
+
+## Standardizing Text Used to Label and Describe Arguments
+
+In the following [PR](https://github.com/pharmaverse/admiral/pull/2065), you will find an example of how the function argument `dataset` was able to be standardized such that the Label and Description of said function argument was aligned across the codebase. 
```

Comment:
Minor, but could you please add something specifying where to look within this PR? Along the lines of "Please see the changes to `derive_adeg_params()`".

Reply:
added